evadb/catalog/catalog_manager.py

```diff
@@ -646,3 +646,6 @@ def get_configuration_catalog_value(self, key: str, default: Any = None) -> Any:
         if table_entry:
             return table_entry.value
         return default
+
+    def get_all_configuration_catalog_entries(self):
```

Comment:
Can we have a type hint for the return value?

Reply:
Sure, will get on this